src/views/InProgress.vue

```diff
@@ -528,8 +528,13 @@ export default defineComponent({
 
       // TODO: update alert message when itemsToReject contains a single item and also in some specific conditions
       let message;
-      if(!outOfStockItem) {
-        message = this.$t('Are you sure you want to perform this action?')
+      if (!outOfStockItem) {
+        const firstItem = itemsToReject[0];
+        if (itemsToReject.length === 1) {
+          message = this.$t('is identified as. This order item will be unassigned from the store and sent to be rebrokered.', { productName: firstItem.productName, rejectReason: firstItem.rejectReason === "REJ_RSN_DAMAGED" ? 'damaged' : (firstItem.rejectReason === "WORN_DISPLAY" ? 'worn display' : 'mismatched')});
```

Comment:
We can make the reject reason description dynamic instead of handling it through conditions.